tests/virtiofs/datavolume.go

```diff
@@ -160,21 +163,43 @@ var _ = Describe("[sig-storage] virtiofs", decorators.SigStorage, func() {
 	})
 
 	Context("VirtIO-FS with an empty PVC", func() {
-
-		var pvc = "empty-pvc1"
+		var (
+			pvc            = "empty-pvc1"
+			originalConfig v1.KubeVirtConfiguration
+		)
 
 		BeforeEach(func() {
 			checks.SkipTestIfNoFeatureGate(virtconfig.VirtIOFSGate)
+			originalConfig = *util.GetCurrentKv(virtClient).Spec.Configuration.DeepCopy()
 			libstorage.CreateHostPathPv(pvc, testsuite.NamespacePrivileged, filepath.Join(testsuite.HostPathBase, pvc))
 			libstorage.CreateHostPathPVC(pvc, testsuite.NamespacePrivileged, "1G")
 		})
 
 		AfterEach(func() {
+			tests.UpdateKubeVirtConfigValueAndWait(originalConfig)
 			libstorage.DeletePVC(pvc, testsuite.NamespacePrivileged)
 			libstorage.DeletePV(pvc)
 		})
 
-		It("should be successfully started and virtiofs could be accessed", func() {
+		It("[serial] should be successfully started and virtiofs could be accessed", Serial, func() {
```

Comment:
Please use uppercase for serial label

```suggestion
		It("[Serial] should be successfully started and virtiofs could be accessed", Serial, func() {
```